ui/src/views/repository-data/components/sync-types-table/components/repository-sync-status.tsx

```diff
@@ -121,30 +122,31 @@ export const RepositorySyncStatus: React.FC<RepositorySyncStatusProps> = (
     >
       {(displayTooltip && tooltipData?.status !== 'empty') && (
         <div
-          ref = {tooltipRef}
-          className={`${
-            displayTooltip ? 'visible' : 'invisible'
-          } absolute z-50 bg-gray-900 rounded text-gray-300 text-sm opacity-80 p-3 whitespace-nowrap`}
+          ref={tooltipRef}
+          className={`${displayTooltip ? 'visible' : 'invisible'
+            } absolute z-50 bg-gray-900 rounded text-gray-300 text-sm opacity-80 p-3 whitespace-nowrap`}
           style={{
             top: eventPosition?.y ? eventPosition?.y - 80 : 0,
             left: eventPosition?.x
-                  ? eventPosition?.x
-                    - ((tooltipRef?.current) ? tooltipRef?.current.clientWidth / 2 : 0)
-                  : 0,
+              ? eventPosition?.x
+              - ((tooltipRef?.current) ? tooltipRef?.current.clientWidth / 2 : 0)
+              : 0,
           }}
         >
           <div className="flex items-center">
             {(tooltipData?.status) && (
               <div className="mr-2">
-                <RepoSyncIcon type={tooltipData?.status as RepSyncStateT}
-                  />
+                <RepoSyncIcon type={tooltipData?.status as RepoSyncStateT}
+                />
               </div>
             )}
             <div>
-              <span className="font-medium text-white mb-0.5">{tooltipData?.status? tooltipData?.status.charAt(0).toUpperCase() + tooltipData?.status.slice(1) : ''}</span>
+              <span className="font-medium text-white mb-0.5">{tooltipData?.status ? tooltipData?.status.charAt(0).toUpperCase() + tooltipData?.status.slice(1) : ''}</span>
               <div className="flex items-center">
-                <span className="text-sm border-r border-gray-600 mr-1.5 pr-1.5 leading-4">3 days ago</span>
-                <span className="text-sm">{tooltipData?.value} min</span>
+                <span className="text-sm border-r border-gray-600 mr-1.5 pr-1.5 leading-4">
+                  {tooltipData?.doneAt ? getTimeAgoFromNow(new Date(tooltipData?.doneAt)) : ''}
+                </span>
+                <span className="text-sm">{tooltipData?.runningTime} seg</span>
```

Comment:
```suggestion
                <span className="text-sm">{tooltipData?.runningTime} sec</span>
```

Should be `sec` for seconds I think. We might want to use one of the functions from `date-fns` here to format the duration?

Reply:
Yeah, you're right, I'm agree with that. Done [fc93831](https://github.com/mergestat/fuse/pull/89/commits/fc93831c5e74d3e72f1a8f6be5660c492e780dc9)